modules/manually-create-identity-access-management.adoc

```diff
@@ -198,8 +198,32 @@ spec:
 ----
 endif::google-cloud-platform[]
 
-. Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.
+. Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. 
 +
+ifdef::ash[] 
+.Sample `CredentialsRequest` object
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
```

Comment:
I extracted CA objects from 4.10/4.11/4.12 nightly build, the yaml file for cluster-image-registry operator is as below:
```
[root@preserve-jima 4.11]# cat 0000_50_cluster-image-registry-operator_01-registry-credentials-request-azure.yaml 
---
apiVersion: cloudcredential.openshift.io/v1
kind: CredentialsRequest
metadata:
  annotations:
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  labels:
    controller-tools.k8s.io: "1.0"
  name: openshift-image-registry-azure
  namespace: openshift-cloud-credential-operator
spec:
  providerSpec:
    apiVersion: cloudcredential.openshift.io/v1
    kind: AzureProviderSpec
    roleBindings:
    - role: Contributor
  secretRef:
    name: installer-cloud-credentials
    namespace: openshift-image-registry
```
The annotation part is missed in doc. It's better to add this part in sample file, because below important warning message also mentions that CA objects with annotation "release.openshift.io/feature-gate: TechPreviewNoUpgrade" should not generate relevant secrets.

Reply:
@jinyunma thank you! great point :) will update

Reply:
@jinyunma updated! Feel free to take a look when you return from PTO :) @bscott-rh this should be ready for peer review soon!